src/main/java/frc/robot/Motion.java

```diff
@@ -0,0 +1,75 @@
+package frc.robot;
+
+public class Motion {
+
+    // Scaler value that will be applied to the power output (significantly less then one)
+    //This Scaler takes velocity and translates it into pwm
+    private double moveConstant;
+
+    // Total time of the move in seconds
+    private double timeMax;
+
+    // Total distance of the move in encoder counts
+    private double distance;
+
+    // The max distance that the encoder value can be away from the specified distance
+    private double tolerance;
+
+    public Motion(double time, double dist, double tol, double moveK){
+        timeMax = time;
+        distance = dist;
+        tolerance = tol;
+        moveConstant = moveK;
+    }
+    
+    /*
+        Returns a double with an output range of [-1.0, 1.0] as the motor power by changing the target distance to be along
+        a polynomial curve according to the current time of the move
+    */
+    public double getPower(double currentPosition, double currentTime) {
+        double targetPosition = distance * polynomial345(currentTime / timeMax);
+        return getFeedbackPower(currentPosition, targetPosition);
+    }
+    
+    private double polynomial345(double time) {
+        return (10 * Math.pow(time, 3.0)) - (15 * Math.pow(time, 4.0)) + (6 * Math.pow(time, 5.0));
+    }
+
+    // Returns a power value between -1.0 and 1.0 for the power of the motor
+    private double getFeedbackPower(double currentPosition, double targetPosition) {
+        double power = moveConstant * Math.sqrt(Math.abs(targetPosition - currentPosition));
+        if (targetPosition - currentPosition < 0.0) {
+            power *= -1.0;
+        }
+        if (power < -1.0) {
+            return -1.0;
+        } else if (power > 1.0) {
+            return 1.0;
+        } else {
+            return power;
+        }
+    }
+    public double getFeedbackPower(double currentPosition, double targetPosition, double moveConstant) {
+        double power = moveConstant * Math.sqrt(Math.abs(targetPosition - currentPosition));
+        if (targetPosition - currentPosition < 0.0) {
+            power *= -1.0;
+        }
+        if (power < -1.0) {
+            return -1.0;
+        } else if (power > 1.0) {
+            return 1.0;
+        } else {
+            return power;
+        }
```

Comment:
This appears to be unused. It should be removed.

If we need it for some reason, the `getFeedbackPower` on line 39 should call it rather than duplicating the code.